atomicapp/cli/main.py

```diff
@@ -9,16 +9,19 @@
 
 from atomicapp import set_logging
 from atomicapp.constants import ANSWERS_FILE, __ATOMICAPPVERSION__, __NULECULESPECVERSION__
+from atomicapp.utils import printStatus
 
 logger = logging.getLogger(__name__)
 
 def cli_install(args):
     install = Install(**vars(args))
     install.install()
+    printStatus("Install Successful.")
```

Comment:
shall we have machine readable and human readable status messages that are send back to cockpit? @stefwalter